### PR TITLE
Convert all quests to revscript

### LINKED ISSUE #45: [to-do] quests to revscriptsys
Migrate all quests to a single file (for each quest) in revscript

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

**Issues addressed:** 
Closes #45

[cont]: https://github.com/EPuncker/orts2/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: data/scripts/orts/quests/hot_cuisine.lua [hunk 1]
```diff
@@ -1,21 +1,27 @@
+local cookBook = Action()
+
 local config = {
 	[1] = "Rotworm Stew\n- 2 Pieces of meat\n- 2 Vials of beer\n- 20 Potatoes\n- 1 Onion\n- 1 Garlic\n- 5 Ounces of flour\n\n",
 	[2] = "Hydra Tongue Salad\n- 2 Hydra tongues\n- 2 Tomatoes\n- 1 Cucumber\n- 2 Eggs\n- 1 Troll green\n- 1 Vial of wine\n\n",
-	[3] = "Roasted Dragon Wings\n- 1 Fresh dead bat\n- 3 Jalape�o peppers\n- 5 Brown breads\n- 2 Eggs\n- 1 Powder herb\n- 5 Red mushrooms\n\n",
+	[3] = "Roasted Dragon Wings\n- 1 Fresh dead bat\n- 3 Jalape�o peppers\n- 5 Brown breads\n- 2 Eggs\n- 1 Powder herb\n- 5 Red mushrooms\n\n",
```
Comment (EPuncker) [inline data/scripts/orts/quests/hot_cuisine.lua:6]: muda todas `ñ` pra `n`

File: data/scripts/orts/quests/the_new_frontier.lua [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+end
+
+function arena.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local player1 = Tile(Position(33080, 31014, 2)):getTopCreature()
+	if not(player1 and player1:isPlayer()) then
+		return true
+	end
+
+	local player2 = Tile(Position(33081, 31014, 2)):getTopCreature()
+	if not(player2 and player2:isPlayer()) then
```
Comment (EPuncker) [inline data/scripts/orts/quests/the_new_frontier.lua:56]:
```suggestion
	if not player2 and player2:isPlayer() then
```

File: data/scripts/orts/quests/the_new_frontier.lua [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+			exitPosition:sendMagicEffect(CONST_ME_TELEPORT)
+		else
+			spectator:remove()
+		end
+	end
+end
+
+function arena.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local player1 = Tile(Position(33080, 31014, 2)):getTopCreature()
+	if not(player1 and player1:isPlayer()) then
```
Comment (EPuncker) [inline data/scripts/orts/quests/the_new_frontier.lua:51]:
```suggestion
	if not player1 and player1:isPlayer() then
```

**Review by torresmon — APPROVED** [review]

File: data/scripts/quests/wrath_of_the_emperor.lua [hunk 4]
```diff
@@ -488,3 +488,307 @@ end
...
+		end
+	elseif item.actionid == 8021 then
+		player:say("Guards heavily patrol this area. Try to stay hidden and do not draw any attention to yourself by trying to attack.", TALKTYPE_MONSTER_SAY)
+	elseif item.actionid == 8022 then
+		if player:getStorageValue(PlayerStorageKeys.WrathoftheEmperor.CrateStatus) ~= 1 then
+			catchPlayer(playerId)
+		end
+	elseif item.actionid == 8023 then
+		-- player:setStorageValue(PlayerStorageKeys.WrathoftheEmperor.CrateStatus, 0)
+		-- doSetCreatureOutfit(cid, {lookTypeEx = 12496}, 1)
```
Comment (torresmon) [inline data/scripts/quests/wrath_of_the_emperor.lua:563]: linhas 562 e 563 deveriam ser comentadas mesmo?

**Review by torresmon — APPROVED** [review]

**Review by torresmon — APPROVED** [review]

**Pull request merged**